recipes/libpq/all/conanfile.py

```diff
@@ -18,7 +18,13 @@ class LibpqConan(ConanFile):
         "with_zlib": [True, False],
         "with_openssl": [True, False],
         "disable_rpath": [True, False]}
-    default_options = {'shared': False, 'fPIC': True, 'with_zlib': True, 'with_openssl': False, 'disable_rpath': False}
+    default_options = {
+        'shared': False,
+        'fPIC': False,
+        'with_zlib': True,
+        'with_openssl': True,
```

Comment:
I dont think we are allowed to change the defaults 🙊 I'll let the CCI team comment

Reply:
Other recipes allowed it:

https://github.com/conan-io/conan-center-index/blob/master/recipes/pcre/all/conanfile.py#L39
https://github.com/conan-io/conan-center-index/blob/master/recipes/librhash/all/conanfile.py#L24
https://github.com/conan-io/conan-center-index/blob/master/recipes/boost/all/conanfile.py#L79

Reply:
In the diff on GitHub it looks like only `openssl` went from False to True. 

The value itself is fine, my concern is that it was changed.

Let's wait to see if the CCI team has a strong opinion.

Reply:
If it's the default value of upstream build system, it's fine.

Reply:
Actually, I checked default value in 13.0, and all options are disabled by default except zlib, so default value of openssl option should be disabled in this recipe.